File: GFG/Array/Day3/MedianOfAnArray.java
import java.util.Arrays;

public class MedianOfAnArray{
    public double findMedian(int[] arr) {
        // Code here.
        Arrays.sort(arr);
        if(arr.length%2==0){
            return (arr[arr.length/2]+arr[(arr.length/2)-1])/2.0;
        }
        return (double)arr[(arr.length)/2];
    }
}

File: GFG/Array/Day3/MedianOfAnArray.py
class Solution:
    def findMedian(self, arr):
        #code here.
        arr.sort()
        if len(arr)%2==0:
            return (arr[len(arr)//2]+arr[(len(arr)-2)//2])/2.0
        else:
            return arr[len(arr)//2]